devito/tools.py

```diff
@@ -56,9 +63,22 @@ def aligned(a, alignment=16):
     extra = alignment / a.itemsize
     buf = np.empty(a.size + extra, dtype=a.dtype)
     ofs = (-buf.ctypes.data % alignment) / a.itemsize
-    aa = buf[ofs:ofs+a.size].reshape(a.shape)
+    aa = buf[ofs:ofs + a.size].reshape(a.shape)
     np.copyto(aa, a)
 
     assert (aa.ctypes.data % alignment) == 0
 
     return aa
+
+
+def get_cache_line_size():
+    """Gets cache line size
+    :returns: cache line size"""
+    if "Linux" in platform.platform():
+        with open('/sys/devices/system/cpu/cpu0/cache/index0/coherency_line_size') as f:
+            return int(f.read())
+    else:  # Running on MAC
+        command = "/usr/sbin/sysctl -a | grep hw.cachelinesize"
```

Comment:
This is just turned out to be a big fucking headache. We couldn't find a portable solution for this. Pretty much *every* modern system has a cache line size of 64 bytes - expect power which is 128 bytes.

I propose we just set it to  64 bytes for now and re-visit when when we integrate a portable solution for abstracting the hardware.